website/src/components/Features.tsx

```diff
@@ -31,6 +57,38 @@ export function Features() {
                 </li>
               ))}
             </ul>
+            {/* email form */}
+            <form className="mt-10 px-20" onSubmit={handleEmailSubmission}>
+              <h3 className="text-lg font-medium tracking-tight text-black">
```

Comment:
**question:** Is there an `h2` prior to this `h3` in the page once rendered ? If not, can you make this an `h2` for semantics ?